red_queen/games/applications/__init__.py

```diff
@@ -49,5 +52,15 @@ def run_qiskit_circuit(benchmark, circuit, backend, optimization_level, shots, e
     else:
         num_1q = op_count.get("sx", 0) + op_count.get("x", 0) + op_count.get("rz", 0)
     info.quality_stats["xi"] = num_2q / (num_1q + num_2q)
-    counts = backend.run(tqc, shots=shots, seed_simulator=123456789).result().get_counts()
+    if marginalize == 0:
```

Comment:
Typically you would use a boolean for this so `if marginalize`, and you default it to `marginalize=False` instead of 0.